build.gradle

```diff
@@ -64,6 +64,8 @@ gitProperties {
   gitPropertiesResourceDir = file("${rootDir}/build")
   extProperty = 'gitProps'
   failOnNoGitDirectory = true
+  keys = ['git.branch', 'git.build.version', 'git.commit.id.abbrev', 'git.commit.id', 'git.commit.message.short',
+          'git.commit.time', 'git.tags']
```

Comment:
The nearest tag seems useful to me as well.

Reply:
sure added.